src/Microsoft.ML.FastTree/FastTree.cs

```diff
@@ -25,6 +25,8 @@
 using Microsoft.ML.Runtime.Training;
 using Microsoft.ML.Runtime.TreePredictor;
 using Newtonsoft.Json.Linq;
+using Microsoft.ML.Core.Data;
+using Microsoft.ML.Runtime.EntryPoints;
```

Comment:
I'm probably just missing something obvious, but why does this now depend on entry-points namespace?

Also sorting.

Reply:
Thank you! Oversight

---
In reply to: [218216150](https://github.com/dotnet/machinelearning/pull/855#discussion_r218216150) [](ancestors = 218216150)